src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,42 @@
+import { useSearchParams } from 'react-router-dom';
+import { Person } from '../types';
+import { PersonLinks } from './PersonLinks';
+import { SearchLink } from './SearchLink';
+import classNames from 'classnames';
+
+type Props = {
+  people: Person[];
+};
+
 /* eslint-disable jsx-a11y/control-has-associated-label */
-export const PeopleTable = () => {
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const getSortIconClass = (field: string) => {
+    if (sort !== field || !sort) {
+      return 'fa-sort';
+    }
+
+    return order === 'desc' ? 'fa-sort-down' : 'fa-sort-up';
+  };
+
+  const sortParams = (value: string) => {
+    if (
+      searchParams.has('sort') &&
+      searchParams.has('order') &&
+      value === sort
+    ) {
+      return { sort: null, order: null };
+    }
+
+    return sort === value
+      ? { sort, order: 'desc' }
+      : { sort: value, order: null };
```

Comment:
The `sortParams` function logic might not correctly toggle the sorting order. When the current sort field is the same as the value, it should toggle between 'asc' and 'desc', but the current logic sets 'desc' directly. Consider revising this logic to correctly toggle the order.